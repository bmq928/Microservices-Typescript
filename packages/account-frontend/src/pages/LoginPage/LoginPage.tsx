import React from 'react'
import { Link } from 'react-router-dom'

import './LoginPage.scss'

export default function LoginPage() {
  return (
    <div className="LoginPage box">
      <div className="field is-horizontal title-group">
        <h4 className="subtitle is-4">Login Page</h4>
      </div>
      <div className="field is-horizontal">
        <div className="field-label is-normal">
          <label className="label">Username:</label>
        </div>
        <div className="field-body">
          <div className="field">
            <div className="control">
              <input
                className="input is-primary"
                type="text"
                placeholder="e.g: Kamejoko"
              />
            </div>
          </div>
        </div>
      </div>
      <div className="field is-horizontal">
        <div className="field-label is-normal">
          <label className="label">Password:</label>
        </div>
        <div className="field-body">
          <div className="field">
            <div className="control">
              <input
                className="input is-primary"
                type="password"
                placeholder="Password"
              />
            </div>
          </div>
        </div>
      </div>
      <div className="field is-horizontal button-group">
        <button className="button is-danger">Cancel</button>
        <button className="button is-primary">Login</button>
        <Link to="/register">Don't have account?</Link>
      </div>
    </div>
  )
}
