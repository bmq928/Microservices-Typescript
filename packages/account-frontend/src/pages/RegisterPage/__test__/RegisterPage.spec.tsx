import React from 'react'
import { shallow } from 'enzyme'

import LoginPage from '../RegisterPage'

describe('[UI] LoginPage', () => {
  it('Match snapshot', () => {
    const wrapper = shallow(<LoginPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
