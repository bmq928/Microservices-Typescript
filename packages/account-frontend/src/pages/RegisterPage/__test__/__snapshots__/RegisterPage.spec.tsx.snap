// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[UI] LoginPage Match snapshot 1`] = `ShallowWrapper {}`;
